File: test/org/rust/lang/core/parser/RustPartialParsingTestCase.java
package org.rust.lang.core.parser;

public class RustPartialParsingTestCase extends RustParsingTestCaseBase {

    public RustPartialParsingTestCase() {
        super("parser/ill-formed");
    }

    public void testFn()        { doTest(true); }
    public void testUseItem()   { doTest(true); }

}

File: test/org/rust/lang/core/parser/RustPartialParsingTestCase.kt
package org.rust.lang.core.parser

class RustPartialParsingTestCase : RustParsingTestCaseBase("parser/ill-formed") {

    // @formatter:off
    fun testFn()                    { doTest(true) }
    fun testUseItem()               { doTest(true) }
    // @formatter:on
}
